hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/CryptoUpdateHandler.java

```diff
@@ -102,13 +103,14 @@ public void preHandle(@NonNull final PreHandleContext context) throws PreCheckEx
         // the target account must sign the transaction
         // 3. If the payer for the transaction is 0.0.2 or system admin, then no signatures are needed for the update
         final var targetAccountKeyMustSign = !waivers.isTargetAccountSignatureWaived(txn, payer);
+
```

Comment:
Curious `calculateFees` method is not implemented in this handler. Could you please create issue for that

Reply:
Created issue #8445 for this.